src/state/content-settings/__tests__/actions.test.js

```diff
@@ -185,4 +235,103 @@ describe('contentSettings thunks', () => {
       done();
     }).catch(done.fail);
   });
+
+  it('sendPostMetadataMap', (done) => {
+    store.dispatch(sendPostMetadataMap('key', 1)).then(() => {
+      expect(postMetadataMap).toHaveBeenCalledWith({ key: 'key', mapping: 1 });
+      const actions = store.getActions();
```

Comment:
It seems to lack length assertion.
```suggestion
      const actions = store.getActions();
      expect(actions).toHaveLength(3);
```